orc8r/cloud/deploy/terraform/orc8r-helm-aws/variables.tf

```diff
@@ -452,3 +452,25 @@ variable "cloudwatch_exporter_enabled" {
   default     = false
   type        = bool
 }
+
+##############################################################################
+# NProbe service variables
+##############################################################################
+
+variable "nprobe_operator_id" {
+  description = "Mobile operator id used by nprobe service."
+  type        = number
+  default     = 1
+}
+
+variable "nprobe_delivery_server" {
+  description = "Remote LIMS address where nprobe service exports records."
+  type        = string
+  default     = "10.10.0.2:6666"
+}
+
+variable "nprobe_skip_verify_server" {
+  description = "allows nprobe service to skip tls server verification when exporting records."
+  type        = bool
+  default     = false
+}
```

Comment:
In my mind, the nprobe architecture is doing too much via *configuration* (infrastructure-level) rather than *in-application* (rest api, nms)

Advantages of the in-application style
- Network operators can make changes without having to redeploy the entire Orc8r
- Nprobe configuration can be scoped to a particular network, allowing multiple operators (sharing the multi-tenant Orc8r) to independently make use of the nprobe functionality

Reply:
@ymasmoudi I wasn't part of the design process as you know, I've been asked to fill in on reviewing these PRs. I think we can come to a compromise between our goals (yours being landing this functionality, mine being keeping Orc8r complexity manageable and removing unnecessary externalities) 🙂 

For the config changes
- From skimming the PR, I don't see where the operator_id is being set -- can you point me in the right direction?
- I strongly think leaving TLS server verification to false is a bad idea. If you accidentally forget to switch it back to true, or even if partners move forward with the feature with an "outdated" Orc8r version, they'll be broadcasting LI data (?) to the world. I can't imagine any feature velocity would be worth that negative security externality

Let me know if you want to schedule a sync to synchronously move through my concerns. I think we're almost there with getting this merged 🙂

Reply:
Since disabling tls server verification is a big concern, I wonder why it is disabled by default in many places in orc8r and feg. 
In addition, all our deployments uses create_application_certs.sh to generate self-signed certs. This script does not use any SNI extension. Thus forcing all integrations with orc8r to disable tls server verification!

Finally, thanks for taking over this review. I'm just not sure if a PR review is the correct place to change a project scope agreed with your team.

Anyhow, all your requested changes were added.

Reply:
For disabling tls server verification: I can't speak for feg, but for orc8r if you follow the thread of each instance of "no tls", it's never enabled by default (as best I can tell)

For self-signed certs -- the server's TLS cert's root is rootCA.pem, which we do in fact add to each gateway so that they can securely connect to orc8rs that use self-signed certs

Anyways, I agree that these requested changes would ideally have come earlier in the design process. I wasn't part of that process, and was asked to help at the last-mile, so that's why we're running with this suboptimal pattern

Thanks for the changes, lgtm 🙂